methodoverride/method_override.py

```diff
@@ -0,0 +1,10 @@
+class ExampleOverride:
+    
+    def __init__(self):
+        self.classAvar = "When init is not overridden"
+        print(self.classAvar)
+
+class ExampleOver(ExampleOverride):
```

Comment:
The file same it is method override but I don't see any example here that is overriding the method.